### ISSUE TITLE
Add Kirins

### ISSUE THREAD

**Issue by mathieucaroff** [ponydevs/pixel.horse#50]
After #23 ,

I'd like Kirins. I'm thinking of Kirins in the style of season 8 episode 23.

## Research

For reference:
https://derpibooru.org/search?q=safe%2C+kirin-ified%2C+score.gt%3A+250
https://derpibooru.org/galleries/11208

### Body

Kirin backs should not be harder to add than Zebra coats.

#### Chest-fluff

I think this one could be added as a new category in the `mane` tab. We could have one or two options here. I'm not investigating what these can be for now. z-index wise, there might be some work to do to make it work with all existing manes.

#### Hooves

Kirin cloven hooves would not be very hard to add.

#### Horn

It would also be nice to have Kirin horns. So far, I'm thinking three kinds of horns:
* either right-left symmetric split (>>1829352 and >>1816265)
* front-back split (>>1998218 >>1874373 >>1832333)
* or not split (>>1860949 >>1832352 >>1855061) -- but it'd still need to be thicker than that of traditional unicorns at the base of the horn.

They'd need the ability to choose the color of the arrows pattern on them. I'm thinking of an auto-color that would take a color lighter than the color of the horn by default. There would be a checkbox to enable picking the color. Then, locking the color would copy the color of the rest of the horn, making thee pattern disappear.

There's probably no need for more than one arrow pattern.

--- Feedback about horns?

#### Rear-legs-back-fluff

Most Kirins have some colored fluff there.

### Already present

**Tail** - The Kirin tails are already there.
**Face** - Kirin have scales on their forehead and on the top of their nose. This is pretty much already supported with `Markings` in the `face` tab. I don't see what more can be done here.

### What I forgot

...if any, should be addressed either by feedback here, or in future fixes I guess.
